FirebasePerformance.podspec

```diff
@@ -62,6 +62,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
   s.dependency 'FirebaseCore', '~> 10.0'
   s.dependency 'FirebaseInstallations', '~> 10.0'
   s.dependency 'FirebaseRemoteConfig', '~> 10.0'
+  s.dependency 'FirebaseSessions', '~> 10.0'
```

Comment:
We should probably actually add a changelog entry for this change. You can write a similar message to https://github.com/firebase/firebase-ios-sdk/pull/10682/files

Reply:
Done